src/main/kotlin/org/rust/ide/intentions/addFmtStringArgument/AddFmtStringArgumentIntention.kt

```diff
@@ -108,7 +108,7 @@ class AddFmtStringArgumentIntention : RsElementBaseIntentionAction<AddFmtStringA
             *newArgs.toTypedArray()
         )
 
-        project.runWriteCommandAction {
+        project.runWriteCommandAction(text, "intention.AddFmStringArgument") {
```

Comment:
Good catch.

Reply:
it's not me, it's the "Hardcoded strings" inspection. I've just added `@NlsContexts.Command` to `runWriteCommandAction`

Reply:
Oh. Wow. That's like a superpower... and absolutely undocumented ((((
In that case, I'll fix it later, together with the other localization changes. It should be a proper localized string and not a `text.capitalize()` hack. Don't want to create merge conflicts now.

Reply:
> Oh. Wow. That's like a superpower... and absolutely undocumented ((((

Yeah. Since IntelliJ team made localization of the platform only recently (and mostly for internal purposes), this subsystem doesn't have enough documentation yet